app/services/auth.js

```diff
@@ -71,8 +71,10 @@ export default Service.extend({
   redirectUrl: null,
 
   signOut(runTeardown = true) {
-    this.localStorage.clearAuthData();
-    this.sessionStorage.clearAuthData();
+    [this.localStorage, this.sessionStorage].map(storage => {
```

Comment:
I'd suggest to use `forEach` here, just to make it explicit that we're operating on the array items, not transforming them.
```suggestion
    [this.localStorage, this.sessionStorage].forEach(storage => {
```
Though it's just an opinion and completely optional, `map` works perfectly as well.